docs/components/MDX/components.tsx

```diff
@@ -144,4 +144,7 @@ const components = {
   }
 }
 
+let currentId = 0
+const uuid = () => `mermaid-${(currentId++).toString()}`
+
```

Comment:
This does not seem to be used anywhere.

Reply:
the Mermaid component has been quashed in one of the merge from main. Added it again